src/Caliburn.Micro.Platform/Platforms/net46-netcore/Bootstrapper.cs

```diff
@@ -83,7 +83,19 @@ public void Initialize()
         protected virtual void StartDesignTime()
         {
             AssemblySource.Instance.Clear();
-            AssemblySource.Instance.AddRange(SelectAssemblies());
+			
+			foreach (var assembly in SelectAssemblies())
+            {
+                try
+                {
+                    if (!AssemblySource.Instance.Contains(assembly))
+                        AssemblySource.Instance.Add(assembly);
+                }
+                catch (ArgumentException)
```

Comment:
If we're checking the assembly source now what times would you expect it to throw this exception?

Reply:
Happens if throwing in all assemblies in the app folder without filtering when publishing self-contained.